queries.sql

```diff
@@ -23,4 +23,68 @@ SELECT * FROM animals
 WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235758216-17ddebe0-05aa-47e9-a0e0-3a341205a6e0.png)

**[OPTIONAL]**

- [ ] It would be nice if you fix these links in your Table of Contents which you can see in the image above to make your readme descriptive and professional 😊